frontend/packages/dev-console/integration-tests/views/git-import-flow.view.ts

```diff
@@ -1,10 +1,15 @@
 /* eslint-disable no-console, promise/catch-or-return */
-import { browser, ExpectedConditions as until, by, element, Key } from 'protractor';
+import { browser, $, ExpectedConditions as until, by, element, Key } from 'protractor';
+import { config } from '@console/internal-integration-tests/protractor.conf';
+const waitForElement = config.jasmineNodeOpts.defaultTimeoutInterval;
+import { enterText, selectByIndex, selectByVisibleText,
```

Comment:
move this relative import statements after absolute imports

Reply:
Sure, I will move this line as part of coding standarda